gatsby-site/src/components/Project/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Link from 'gatsby-link'
+
+export default function Project(props) {
+  return(
+      <a>
+        <img src={props.logo} alt={props.name} className="project_logo" />
```

Comment:
This looks great!  I'd suggest changing this class name, though.  When we change things for the new landing-page designs, the project component will have a background image, a wavy svg, and a logo (aka, icon and name pair), so this could get confusing later.  Maybe call it project__background-image?